osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -31,6 +32,8 @@ public class DrawableSpinner : DrawableOsuHitObject
 
         private readonly IBindable<Vector2> positionBindable = new Bindable<Vector2>();
 
+        private bool spinnerFrequencyModulate;
```

Comment:
I believe this shouldn't reside in the main `DrawableSpinner`, but in the legacy spinner implementations.

Reply:
not sure about this one. also though so but would likely involve moving the sample playback completely. maybe something which can be done at a later point.